code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm

```diff
@@ -159,6 +159,24 @@
 	if(prob(5))
 		M.vomit()
 
+/datum/reagent/toxin/expired_medicine
+	name = "Expired Medicine"
+	id = "expired_medicine"
+	description = "Some form of liquid medicine that is well beyond its shelf date. Administering it now would cause illness."
+	taste_description = "bitterness"
+	reagent_state = LIQUID
+	strength = 5
+
+/datum/reagent/toxin/expired_medicine/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
```

Comment:
Why is this reagent set up like this, exactly?

Reply:
I'm not sure what you're asking. To my knowledge its set up like most other reagents are.

Reply:
If I'm reading this correctly, if they inject the med they take double the usual damage because of the super, and they have a chance to vomit.
But if they inject it, they take only 2/3rds of the damage they'd take from a normal strength 5 toxin.

Reply:
If I'm reading this correctly, if they inject the med they take double the usual damage because of the super, and they have a chance to vomit.
But if they inject it, they take only 2/3rds of the damage they'd take from a normal strength 5 toxin.

Reply:
If I'm reading this correctly, if they inject the med they take double the usual damage because of the super, and they have a chance to vomit.
But if they inject it, they take only 2/3rds of the damage they'd take from a normal strength 5 toxin.

Reply:
The super's there because of bad copypasta and I'll go fix it and fix the origin of it as well (the mold reagent). The 2/3rd interaction is when it gets into the mob's stomach somehow.

Reply:
Are there not pills of the stuff?

Also, reagents already naturally do something when ingested, they transfer slowly to the bloodstream, unless you specifically want this one to do less damage instead.

Reply:
This PR doesn't add pills of any kind. Added the stomach thing as a future thing in-case someone wants to add fifty year old pills, based on some other reagents that appeared to transfer from the stomach to bloodstream like mutagen.

I can remove the override on this if desired since I didn't realize the reagent system already did that for you if you don't override.

Reply:
Think it might be better to futureproof this for when we do start adding more pills.